.github/workflows/build-rd-cpp.yml

```diff
@@ -0,0 +1,204 @@
+name: Build RD CPP
```

Comment:
Sync name with rd-net part please

File names also not in sync. I named rd-net build as `build.yaml` as I expected configuration for a different parts will be located in the same file. As we have `rd-cpp.yml` it seems logical to rename `build.yml` to `build-rd-net.yml`